### ISSUE TITLE
bias defaults to True for nn.Conv2d

### ISSUE THREAD

**Issue by jacobgorm** [huawei-noah/VanillaNet#32]
Thanks for publishing this quite interesting work. I believe it is customary to use either bias or BatchNorm, but not both at once. the nn.Conv2d layers not have bias=False added to their constructor args, as otherwise they would default to bias=True?

**Comment by HantingChen** [comment]
Thanks for the nice concern. Since the BatchNorm will normalize the input features, it is almost the same for setting bias=True and bias=False. Therefore, both settings are fine.
